tests/cases/compiler/exportDefaultInterfaceClassAndValue.ts

```diff
@@ -0,0 +1,4 @@
+const foo = 1
+export default foo
+export default class Foo {}
+export default interface Foo {}
```

Comment:
can you please add tests for

- non-default exports,
- export list with duplicate entries
- a file without `interface Foo`, since just const and class should be enough to give an error
- a version with type alias instead of interface
- actually, several different combinations of interfaces and other values/types/namespaces